deploy.sh

```diff
@@ -15,6 +15,8 @@ if [ -z ${D4M_HOME+x} ]; then
   echo "Not installing Graphulo JAR in D4M installation because D4M_HOME is not set";
 else
   cp $(echo target/graphulo-*.jar | tr ' ' '\n' | grep -v alldeps) "$D4M_HOME/lib"
+  cp -f target/graphulo-*-alldeps.jar $D4M_HOME/lib
```

Comment:
Confirmed, this isn't needed here. It could be put there just so it's easy to find when you want to add it to Accumulo (ex: I build graphulo and d4m in my home directory, but only the final d4m_api folder is moved to a shared location)? That'd be my guess.